tutorials/core-concepts/pages/pages.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Page Templates
+layout: tutorial
+---
+
+# Pages
+
+## How to create new pages
+
+In Apostrophe, new pages are created from the page administration interface. When you create a new page, you can select from any of the available templates.
+
+Out of the box, Apostrophe provides one template, but you can create any number of custom templates to meet your needs.
+
+
+### How to create page templates
+
+The page template determines the page’s layout and what kind of content can be added to a page. You can create a new page template in two steps:
+
+1. Create a new template in `lib/modules/apostrophe-pages/views/pages` which contains the code to define your page layout.
+
+![](/.gitbook/assets/pages-directory-tree.png)
+
+2. Register the template in `app.js`
+
+{% hint style="info" %}
+Apostrophe uses the [Nunjucks templating language](https://mozilla.github.io/nunjucks/). Nunjucks uses a mix of standard HTML and CSS with Nunjucks specific tags. You can learn more about using Nunjucks with Apostrophe in the  [Nunjucks Helper Functions](/tutorials/core-concepts/working-with-templates/nunjucks-helper-functions.md) and [Nunjucks Filters](/tutorials/core-concepts/working-with-templates/nunjucks-filters.md) references.
```

Comment:
This makes it sound like Nunjucks has an opinion about HTML and CSS. It doesn't, it's just a template language for outputting text. I would say something like "Nunjucks allows you to write your HTML markup without repeating yourself, and makes it easy to insert dynamic content like page titles and editable content areas."